interpreter/python/python.go

```diff
@@ -847,3 +837,26 @@ func Loader(ebpf interpreter.EbpfHandler, info *interpreter.LoaderInfo) (interpr
 
 	return pd, nil
 }
+
+func findInterpreterRanges(info *interpreter.LoaderInfo) ([]util.Range, error) {
```

Comment:
```suggestion
func findInterpreterRanges(info *interpreter.LoaderInfo) (interpRanges []util.Range, err error) {
```
I think the new lint will complain about wanting named return variables (if there's more than one like here). Naming them here also allows removing the `var` declarations of these names below.